src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/TestFactory.cs

```diff
@@ -448,12 +449,13 @@ private void AddHackerUser(List<ProCoSysProject> commonProCoSysProjects)
                         Permissions.USER_READ,
                         Permissions.IPO_READ,
                         Permissions.IPO_SIGN,
-                        Permissions.IPO_WRITE
+                        Permissions.IPO_WRITE,
+                        Permissions.IPO_CREATE
```

Comment:
why should accepter have create permissions?

Reply:
I guess in my head accepter, planner, and completed could be the same person - but I see now that for testing purposes it shouldnt be like that

Reply:
Exactly. Then we have to use planner test user to test create IPO endpoint. Other should get access denied